InterfacePython/TasmanianSG.py

```diff
@@ -713,6 +714,34 @@ def makeFourierGrid(self, iDimension, iOutputs, iDepth, sType, liAnisotropicWeig
 
         pLibTSG.tsgMakeFourierGrid(self.pGrid, iDimension, iOutputs, iDepth, c_char_p(sType), pAnisoWeights, pLevelLimits)
 
+    def makeGlobalGridCustom(self, iDimension, iOutputs, iDepth, sType, iCustomTabulated, liAnisotropicWeights=[], liLevelLimits=[]):
```

Comment:
This should read `pCustomTabulated` when you prefix with `i` it leads people to think it is an integer.

Reply:
Done.